kraken/lib/src/dom/tags/img.dart

```diff
@@ -294,6 +274,19 @@ class ImageElement extends Element {
     }
   }
 
+  // Delay image size setting to next frame to make sure image has been layouted
+  // to wait for percentage size to be caculated correctly in the case of image has been cached
+  bool _hasImageLayoutCallbackPending = false;
+  void _handleImageResizeAfterLayout () {
```

Comment:
There should be no space between the name of a function and the (left parenthesis) of its parameter list.